src/lua/uri.lua

```diff
@@ -212,10 +212,15 @@ local function params(opts)
     return query_string
 end
 
+local function values(...)
+    return {...}
+end
+
 return {
     parse_many = parse_many,
     parse = parse,
     format = format,
+    values = values,
```

Comment:
Regarding the description and the test (`uri.values()` commit).

The function is public and recommended for usage together with the http client (and the uri module I guess). The documentation request said the opposite and it is incorrect.

A return value of `uri.values()` should be considered as opacue for a user (we can change it to userdata for example): the only what matter is how it is encoded into the URL. There is no sense to document and test how `{...}` works, only URI encoding matters.

BTW, examples in YAML are hard to understand. For example, can you tell for sure what the following value is?

```
---
- -
...
```

How to express `<...>?a`, `<...>?a=`? How `uri.values()` and `uri.values('')` are encoded into an URI? The documentation request doesn't answer those questions.